Small readability improvements on issue templates (#1228)
jdm-contrib/jdm: .github/ISSUE_TEMPLATE/update_entry.md

---

> name: Update Entry.

Reviewer: ```suggestion
name: Update Entry
```
No need for that in a title

Author: Beg to differ on that, but i can live with it.

> about: Tell us if a service you know about has change it deletion process.

Reviewer: ```suggestion
about: Tell us if a service you know about has changed its deletion process.
```

Author: Thanks for correcting me again.

> about: Tell us if a service you know about has changed its deletion process.
> title: 'Update (the service's Name).'

Reviewer: ```suggestion
about: Tell us if a service you know about has changed its deletion process
title: 'Update (the service's Name)'
```

## Data

- `name`: <!-- New Name -->

> - `url`: <!-- New Url -->

Reviewer: ```suggestion
- `url`: <!-- New URL -->
```

- `difficulty`: <!-- New Difficulty: easy/medium/hard/impossible -->
- `notes`: <!-- New Notes -->
- `email`: <!-- New Email Address -->
- `email_subject`: <!-- New Email Subject -->
- `email_body`: <!-- New Email Body -->

> - `domains`: <!-- New Domains for out [Browser Extensions](https://github.com/jdm-contrib). -->

Reviewer: Why removing the checkboxes?

Author: Most people seem to remove them, and besides, what's the use case even for them?

Reviewer: Sounded easier to just check what changed, but I guess leaving it blank has the same meaning, so let's try that out instead then.

There are more complex issue templates now, but I don't have the time to play with them nor I think it will be a great improvement for us here.

Author: "Don't fix it if it ant broke"
